src/Microsoft.PowerShell.ConsoleHost/host/msh/CommandLineParameterParser.cs

```diff
@@ -578,9 +722,9 @@ private void ParseHelper(string[] args)
                 else if (MatchSwitch(switchKey, "wait", "w"))
                 {
                     // This does not need to be localized: its chk only 
-
-                    _ui.WriteToConsole("Waiting - type enter to continue:", false);
-                    _ui.ReadLine();
+                    
+                    ((ConsoleHostUserInterface)_hostUI).WriteToConsole("Waiting - type enter to continue:", false);
```

Comment:
Then this will fail.  Note that this is for debugging interactive console only and should never be called without ConsoleHostUserInterface.

---
In reply to: [85439307](https://github.com/PowerShell/PowerShell/pull/2538#discussion_r85439307) [](ancestors = 85439307)